app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -11,7 +11,8 @@ def create
         if @book_suggestion.save
           render json: @book_suggestion, status: :created
         else
-          render json: { errors: @book_suggestion.errors.full_messages }
+          render json: { errors: @book_suggestion.errors.full_messages },
+                 status: :internal_server_error
```

Comment:
Don't send an :internal_server_error because this is the default error if your API failed because of X reason (like dividing by 0), send a :bad_request instead because its a bad_request to your API

Reply:
Wait, if the error is because the product doesn't have correct values and it doesn't pass the validations the status code should be unprocessable_entity (422)

Reply:
Yeah, sorry my bad @alanlamaswolox